app/code/Transoft/Blog/Controller/Adminhtml/Post/Save.php

```diff
@@ -0,0 +1,75 @@
+<?php
+declare(strict_types=1);
+namespace Transoft\Blog\Controller\Adminhtml\Post;
+
+use Magento\Backend\App\Action;
+use Magento\Backend\App\Action\Context;
+use Magento\Backend\Model\Session;
+use Magento\Framework\App\Action\HttpPostActionInterface;
+use Magento\Framework\View\Result\PageFactory;
+use Transoft\Blog\Model\PostFactory;
+
+/**
+ * Class Save
+ *
+ * Save controller class
+ */
+class Save extends Action implements HttpPostActionInterface
+{
+    const ADMIN_RESOURCE = 'Post';
+
+    protected $resultPageFactory;
+    protected $postFactory;
```

Comment:
Why is it **protected**? All non-public properties and methods SHOULD be private

Reply:
fixed